### PR TITLE
Kia Niro PHEV 2018: add missing FW versions

### PR DESCRIPTION
**Pull request by bnoeldner**
Fingerprint for Kia Niro PHEV 2018

Test Route ID
2dc37a5ccd0b2813|2022-07-14--16-11-57

### CODE REVIEW

**Comment by sshane** [comment]
Can you confirm your minimum enable (cruise control) speed?

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -984,20 +984,24 @@ class Buttons:
...
+      b'\xf1\x816H6F4051\x00\x00\x00\x00\x00\x00\x00\x00',
+      b'\xf1\x816H6D1051\x00\x00\x00\x00\x00\x00\x00\x00',
     ],
     (Ecu.transmission, 0x7e1, None): [
-      b"\xf1\x816U3J2051\000\000\xf1\0006U3H0_C2\000\0006U3J2051\000\000PDE0G16NS2\xf4\'\\\x91",
-      b'\xf1\x816U3J2051\000\000\xf1\0006U3H0_C2\000\0006U3J2051\000\000PDE0G16NS2\000\000\000\000',
+      b"\xf1\x816U3J2051\x00\x00\xf1\x006U3H0_C2\x00\x006U3J2051\x00\x00PDE0G16NS2\xf4'\\\x91",
+      b'\xf1\x816U3J2051\x00\x00\xf1\x006U3H0_C2\x00\x006U3J2051\x00\x00PDE0G16NS2\x00\x00\x00\x00',
+      b'\xf1\x816U3H3051\x00\x00\xf1\x006U3H0_C2\x00\x006U3H3051\x00\x00PDE0G16NS1\x00\x00\x00\x00'
+      b'\xf1\x816U3H3051\x00\x00\xf1\x006U3H0_C2\x00\x006U3H3051\x00\x00PDE0G16NS1\x13\xcd\x88\x92',
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:994]: Do these two fw versions switch between drives?

Comment (sshane) [reply selfdrive/car/hyundai/values.py:994]: Ah I see it does from your Discord comment

**Comment by bnoeldner** [comment]
> 
Minimum enable Cruise Control speed on the 2018 Kia Niro is 20MPH.  SCC will disengage once it is below 9mph.

**Pull request merged**